src/App.js

```diff
@@ -31,24 +30,32 @@ function App() {
   // returns an array with 2 elements: store (initial state) & dispatch to handle state
   const [store, dispatch] = useReducer(reducer, initialState);
 
-  // loads initialWineListings and initialCommentList in componentDidMount
+  // loads initialCommentList in componentDidMount
   useEffect(() => {
-    getWineListings()
-    .then(wineListings => {
+    getComments().then((commentsList) => {
+      dispatch({
+        type: "setAllComments",
+        data: commentsList,
+      });
+    });
+    // dispatch({
+    //   type: "setAllComments",
+    //   data: initialCommentList,
+    // });
+  }, []);
+
+  // loads initialWineList in componentDidMount
+  useEffect(() => {
+    getWineListings().then((wineListings) => {
       dispatch({
         type: "setWineListings",
-        data: wineListings
-      })
-    })
-    // fetch
+        data: wineListings,
```

Comment:
I believe you need to remove the comma at end of this line here. Not sure if this makes any difference to your code.

Reply:
that one automatically adds into the code with the prettier extension